### PR TITLE
Make kill confirm message not ephemeral

### PR DESCRIPTION
**Pull request by CarrotManMatt**

### CODE REVIEW

File: cogs/source.py [hunk 1]
```diff
@@ -20,8 +20,9 @@ async def source(self, ctx: TeXBotApplicationContext) -> None:
         """Definition & callback response of the "source" command."""
         await ctx.respond(
             (
-                "TeX is an open-source project made specifically for the CSS Discord server! "
-                "You can see and contribute to the source code at https://github.com/CSSUoB/TeX-Bot-Py-V2"
+                "TeX-Bot is an open-source project "
+                "made specifically for the CSS Discord server!\n"
+                "You can see and contribute to the source code at [CSSUoB/TeX-Bot-Py-V2](https://github.com/CSSUoB/TeX-Bot-Py-V2)."
```
Comment (MattyTheHacker) [inline cogs/source.py:25]: This change is outside the scope of this pr 😩

File: cogs/ping.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ async def ping(self, ctx: TeXBotApplicationContext) -> None:
             random.choices(
                 [
                     "Pong!",
-                    "64 bytes from TeX: icmp_seq=1 ttl=63 time=0.01 ms",
+                    "64 bytes from TeX-Bot: icmp_seq=1 ttl=63 time=0.01 ms",
```
Comment (MattyTheHacker) [inline cogs/ping.py:26]: out of scope :(

**Review by MattyTheHacker — CHANGES_REQUESTED** [review]

**Comment by CarrotManMatt** [comment]
Let me have my fun, boy!

**Review by MattyTheHacker — APPROVED** [review]
absolutely disgraceful you should be ashamed of yourself

**Comment by CarrotManMatt** [comment]
I'm not ashamed of me but my mum is.
